assets/sass/components/_content.scss

```diff
@@ -0,0 +1,56 @@
+article {
```

Comment:
@nucro please use the "@include fontSize" mixin for all font sizes.  please reference this file for an example:
https://github.com/moxienyc/amtrak-journey/blob/develop/wp-content/themes/amtrak-journey/assets/sass/modules/_blog-feed.scss#L43